src/components/05_pages/Content/Content.js

```diff
@@ -79,11 +80,17 @@ class Content extends Component {
   state = {
     contentTypes: [],
     status: null,
+    sort: { path: 'changed', direction: 'DESC' },
```

Comment:
We probably should use the redux state for storing the sort settings since it's relevant information related to the data.

Reply:
My only concern is that this information currently is only used for this page. Keeping it in Redux adds complexity that we don't currently need.

Reply:
Well with the same argument we could argue that the content should not be in the redux store either :)

Reply:
Yeah. I agree. I think maybe we should make a _followup_ to try and decide what types of information we store in redux vs. what is kept in the component.